cf-agent/verify_files.c

```diff
@@ -991,8 +991,46 @@ PromiseResult FindAndVerifyFilesPromises(EvalContext *ctx, const Promise *pp)
 
 /*****************************************************************************/
 
+static DefineClasses GetExpandedClassDefinitionConstraints(const EvalContext *ctx, const Promise *pp)
+{
+    const char *namespace = PromiseGetBundle(pp)->ns;
```

Comment:
Should there be an `assert(pp != null);` here?

Reply:
Ah, I see you do it down below in the exposed function and not in this static function. :+1: